static/app/components/inputGroup.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import Button from 'sentry/components/button';
+import {
+  Input,
+  InputGroup,
+  InputLeadingItems,
+  InputTrailingItems,
+} from 'sentry/components/inputGroup';
+
+describe('InputGroup', function () {
```

Comment:
Do we want any snapshot tests?

Reply:
Oh true https://github.com/getsentry/sentry/pull/40270/commits/797818c200ebd4eac135bb57134d40c5a34bab22